CONTRIBUTING.md

```diff
@@ -0,0 +1,20 @@
+Issues
+------
+
+If you've found a bug or have a great idea for a new feature that will help all users of impress.js, let us know by [adding your suggestion]
+(http://github.com/bartaz/impress.js/issues/new) to the [issues list](https://github.com/bartaz/impress.js/issues).
+
+Guidelines:
+
+* If reporting a bug, please add a [simplified example](http://sscce.org/).
```

Comment:
This is very useful to prevent Pull Requests or issues that don't make it clear what the problem is. In my experience with popular open-source projects, this is not read by everyone, but at least serves as an argument to request for additional information or even close a request in which we can't do anything about it.